rocky/Makefile

```diff
@@ -34,17 +34,20 @@ run:
 test:
 	python3 manage.py test
 
-utest: ## Run the integration tests.
-	docker compose -f base.yml  -f .ci/docker-compose.yml kill
-	docker compose -f base.yml  -f .ci/docker-compose.yml down --remove-orphans
-	docker compose -f base.yml -f .ci/docker-compose.yml build
-	docker compose -f base.yml  -f .ci/docker-compose.yml run --rm rocky_tests
-
-itest: ## Run the integration tests.
-	docker compose -f base.yml  -f .ci/docker-compose.yml kill
-	docker compose -f base.yml  -f .ci/docker-compose.yml down --remove-orphans
-	docker compose -f base.yml -f .ci/docker-compose.yml build
-	docker compose -f base.yml  -f .ci/docker-compose.yml run --rm rocky_integration_tests
+testclean:
+	docker compose  -f .ci/docker-compose.yml kill
+	docker compose  -f .ci/docker-compose.yml down --remove-orphans
+	docker compose -f .ci/docker-compose.yml build
+
+utest: testclean ## Run the unit tests.
+	docker compose  -f .ci/docker-compose.yml run --rm rocky_tests
+
+itest: testclean ## Run the integration tests.
+	docker compose  -f .ci/docker-compose.yml run --rm rocky_integration
+
+bench: testclean ## Run the report benchmark.
+	docker compose  -f .ci/docker-compose.yml run --rm rocky_integration \
+	python -m cProfile -o .ci/bench_$$(date +%Y_%m_%d-%H:%M:%S).pstat -m pytest -m slow --no-cov tests/integration
```

Comment:
Nice one, a good example on how to use task dependencies. I would only suggest `clean` as an alternative task name, but I'll leave it up to you.

Reply:
Could be a bit confusing shadowing the main `make clean` we have in the base directory I think